### PR TITLE
fix: fix getting RMC bundle (SDKCF-6751)

### PR DESCRIPTION
**Pull request by Esakkiraja-Pothikannan**
# Description
- Initializing the RMC bundle using the Bundle URL

## Links
- SDKCF-6751

# Checklist
- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
- [ ] I have added to the [changelog](../blob/master/CHANGELOG.md#Unreleased)
- [ ] I wrote/updated tests for new/changed code
- [ ] I removed all sensitive data **before every commit**, including API endpoints and keys, and internal links
- [ ] I ran `fastlane ci` without errors
- [ ] All project file changes are replicated in `SampleSPM/SampleSPM.xcodeproj` project

### CODE REVIEW

**Review by SoumenRautray — APPROVED** [review]

File: Sources/RInAppMessaging/Utilities/BundleInfo.swift [hunk 1]
```diff
@@ -68,7 +68,11 @@ internal extension Bundle {
     }
 
     static var rmcResources: Bundle? {
-        .init(identifier: "RMC-RMC-resources")
+        guard let rmcBundleUrl = main.resourceURL?.appendingPathComponent("RMC_RMC.bundle"),
+        let bundle = Bundle(url: rmcBundleUrl) else {
```
Comment (anish-munirathinam) [inline Sources/RInAppMessaging/Utilities/BundleInfo.swift]: nit: indent this line

**Review by anish-munirathinam — APPROVED** [review]

File: Sources/RInAppMessaging/Utilities/BundleInfo.swift [hunk 2]
```diff
@@ -68,7 +68,11 @@ internal extension Bundle {
     }
 
     static var rmcResources: Bundle? {
-        .init(identifier: "RMC-RMC-resources")
```
Comment (imaginaris) [inline Sources/RInAppMessaging/Utilities/BundleInfo.swift:71]: I don't know how this stopped working 🤔

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,6 +9,7 @@
 	- Added rmcsdk version parameter to all api calls [SDKCF-6709]
 - Fixes:
 	- Fixed Xcode 15 beta errors [SDKCF-6692]
+	- Fixed Finding RMC Bundle [SDKCF-751]
```
Comment (imaginaris) [inline CHANGELOG.md]: 751?

File: Sources/RInAppMessaging/Utilities/BundleInfo.swift (hunk 2 above)
Comment (Esakkiraja-Pothikannan) [reply Sources/RInAppMessaging/Utilities/BundleInfo.swift:71]: Not sure, what went wrong now.

**Pull request merged**